modules/explorer-core/src/main/scala/org/ergoplatform/explorer/db/repositories/AssetRepo.scala

```diff
@@ -36,6 +37,16 @@ trait AssetRepo[D[_], S[_[_], _]] {
     offset: Int,
     limit: Int
   ): S[D, Address]
+
+  /** Get boxes where all tokens where issued
+    * according to EIP-4 https://github.com/ergoplatform/eips/blob/master/eip-0004.md
+    */
+  def getAllIssuingBoxes: D[List[ExtendedOutput]]
```

Comment:
I think it's better to wrap the result into streaming effect here, as the result set can potentially not fit into list.
`S[D, ExtendedOutput]`

Reply:
Done.